percona-server-8.0/ps-entry.sh

```diff
@@ -248,7 +245,15 @@ else
   CALL_HOME_OPTIONAL_PARAMS+=" -c 2"
 fi
 
+
 # PERCONA_TELEMETRY_DISABLE is handled at the very beginning of call-home.sh
 /call-home.sh -f "PRODUCT_FAMILY_PS" -v "${PS_TELEMETRY_VERSION}" -d "DOCKER" ${CALL_HOME_OPTIONAL_PARAMS} &> /dev/null || :
```

Comment:
As we spoke on Slack, we can move this line to the beginning on `telemetry-agent-supervisor.sh`. This way will speed up container start a bit. (remember to export `CALL_HOME_OPTIONAL_PARAMS`)

Reply:
CALL_HOME_OPTIONAL_PARAMS is declared as an ENV variable in Dockerfile` ENV CALL_HOME_OPTIONAL_PARAMS=" -s ${OS_VER}"` ,so we need not export it separately. Verified this by running the container with the changes.